Makefile.dev.example

```diff
@@ -0,0 +1,66 @@
+# A quick and conveniant way to use this Makefile is by creating a
+# make shell alias like the following:
+#
+# alias saleor='make -f Makefile.dev.example'
+#
+# then use it as you would make:
+#
+# saleor <target>
+
+# --build-arg=SKIP_LOCK=$(SKIP_LOCK) can be added to the 
+# image-build target below to build with latest pipfile.
+# SKIP_LOCK=--skip-lock
+HOST_UID=1000 # Change this to your systems userid
```

Comment:
@bennylb 

How about this instead?

```
HOST_UID="$(id -u)"
HOST_UID=${HOST_UID:-1000}
HOST_GID="$(id -g)"
HOST_GID=${HOST_GID:-1000}
```

Reply:
Yeah this is a good idea. I'll make the change.